lib/runners/objc.js

```diff
@@ -1,37 +1,40 @@
 var shovel = require('../shovel'),
     util = require('../util'),
+    temp = require('temp'),
     path = require('path'),
-    temp = require('temp');
+    exec = require('child_process').exec;
+
+function compile(args, cb) {
+    args.unshift('gcc', '`gnustep-config --objc-flags`');
+    exec(args.join(' '), cb);
+}
+
 
 module.exports.run = function run(opts, cb) {
-    // Sandbox expects files to have this directory as an ancestor
-    var objcDir = '/tmp/objc';
-    util.mkdirParentSync(objcDir);
     temp.track();
-    var dir = temp.mkdirSync({dir: objcDir}),
-	      objcSandbox = path.resolve('frameworks', 'osx', 'objc.sb');
+    var dir = temp.mkdirSync('objc');
+    var args = [];
 
     shovel.start(opts, cb, {
         solutionOnly: function (runCode, fail) {
-            var solutionFile = util.codeWriteSync('objc', opts.code, dir, 'main.m'),
-                command = 'clang -lobjc -framework Foundation -o main ' + solutionFile;
-
-            if ("setup" in opts) {
```

Comment:
Yeah, I saw something like that .. Currently now Objective C don't really need additional file since it just a core header to import and the main function ..